### DISCUSSION TITLE
Emission in AR/model viewer  ?

### DISCUSSION THREAD

**Discussion by rianlukog** [google/model-viewer#2536] [Q&A] [answered]
Is it possible for Ar and 3d viewports to support the emission of objects ? 
A lit electric bulb for example

**Comment by elalish** [comment, marked as answer]
Yes, you can try it by editing emission factor on the materials panel of our editor. Of course, it will not illuminate other objects in the scene, as is the case with most real-time renderers.

**Reply by rianlukog** [reply]
I do have an object that has an emission pre-applied when I export it as USDZ / glb it doesn't seem to be emissive, any tips?

**Reply by rianlukog** [reply]
Yes, found by selecting the emission material and changed the color from black to white and value
